src/main/java/frc/robot/subsystems/swerve/modules/Module.java

```diff
@@ -20,24 +21,26 @@
 public class Module {
 
     private final ModuleInputsContainer moduleInputsContainer;
-    private final ModuleUtils.ModuleName moduleName;
+    private final ModuleUtils.ModulePosition modulePosition;
     private final ISteer iSteer;
     private final IDrive iDrive;
     private final IEncoder iEncoder;
     private final ModuleConstants constants;
+    private final String logPath;
 
     private SwerveModuleState targetState;
     private Rotation2d startingSteerAngle;
     private boolean isClosedLoop;
 
-    public Module(ModuleUtils.ModuleName moduleName, ModuleConstants constants, IEncoder iEncoder, ISteer iSteer, IDrive iDrive) {
-        this.moduleName = moduleName;
+    public Module(ModuleUtils.ModulePosition modulePosition, ModuleConstants constants, IEncoder iEncoder, ISteer iSteer, IDrive iDrive) {
```

Comment:
```suggestion
    public Module(ModuleConstants constants, IEncoder iEncoder, ISteer iSteer, IDrive iDrive) {
```
insert the fully log path to the ModuleConstants in the factory

Reply:
I can delete the class variable, but I can't delete the parameter because I need it for the log path

Reply:
I want the constants to contain the fully log path, prefix + modulePosition